<html>
  <head>
    <meta charset="utf-8">
    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
    <link rel="stylesheet" type="text/css" href="https://2020.igem.org/wiki/index.php?title=Template:Korea-SIS/CSS&action=raw&ctype=text/css" />
  </head>
  <body>
    <header>
      <div class="header-inner">
        <a class="logo-set" href="https://2020.igem.org/Team:Korea-SIS">
          <div class="logo-team">
            <div>
              <img class="logo" src="https://2020.igem.org/wiki/images/3/33/T--Korea-SIS--logo.png"/>
            </div>
            <div class="teamname">KOREA SIS</div>
          </div>
        </a>
        <div class="navbar">
          <div class="main-menu">
            <div class="sub-menu">
              <a class="without-content" href="https://2020.igem.org/Team:Korea-SIS" onmouseover="this.style.backgroundColor='#C9D787'" onmouseout="this.style.backgroundColor='#7D8A2E'">
                Home&nbsp;
              </a>
            </div>
            <div class="sub-menu">
              <a class="with-content">Team</a>
              <div class="sub-menu-content">
                <div><a href="https://2020.igem.org/Team:Korea-SIS/Members">Members</a></div>
                <div><a href="https://2020.igem.org/Team:Korea-SIS/Collaborations">Collaborations</a></div>
                <div><a href="https://2020.igem.org/Team:Korea-SIS/Attributions">Attributions</a></div>
              </div>
            </div>
            <div class="sub-menu">
              <a class="with-content">Project</a>
              <div class="sub-menu-content">
                <div><a href="https://2020.igem.org/Team:Korea-SIS/Description">Description & Inspiration</a></div>
                <div><a href="https://2020.igem.org/Team:Korea-SIS/Engineering">Engineering</a></div>
                <div><a href="https://2020.igem.org/Team:Korea-SIS/Model">Project Modeling</a></div>
                <div><a href="https://2020.igem.org/Team:Korea-SIS/Implementation">Proposed Implementation</a></div>
                <div><a href="https://2020.igem.org/Team:Korea-SIS/Labnotes">Labnotes</a></div>
                <div><a href="https://2020.igem.org/Team:Korea-SIS/Contribution">Contribution</a></div>
              </div>
            </div>
            <div class="sub-menu">
              <a class="without-content" href="https://2020.igem.org/Team:Korea-SIS/Human_Practices" onmouseover="this.style.backgroundColor='#C9D787'" onmouseout="this.style.backgroundColor='#7D8A2E'">
                Human Practices&nbsp;
              </a>
            </div>
            <div class="sub-menu">
              <a class="without-content" href="https://2020.igem.org/Team:Korea-SIS/Education" onmouseover="this.style.backgroundColor='#C9D787'" onmouseout="this.style.backgroundColor='#7D8A2E'">
                Education&nbsp;
              </a>
            </div>
            <div class="sub-menu">
              <a class="without-content" href="https://2020.igem.org/Team:Korea-SIS/Judging" onmouseover="this.style.backgroundColor='#C9D787'" onmouseout="this.style.backgroundColor='#7D8A2E'">
                Judging Form&nbsp;
              </a>
            </div>
            <div class="sub-menu">
              <a class="without-content" href="https://2020.igem.org/Team:Korea-SIS/Safety" onmouseover="this.style.backgroundColor='#C9D787'" onmouseout="this.style.backgroundColor='#7D8A2E'">
                Safety&nbsp;
              </a>
            </div>  
          </div>
        </div>
      </div>
    </header>

    <nav>
      <div class="nav-menus"> 
        <div class="nav-a"><a href="javascript:window.scrollToId('method')">Method of Promotion</a></div>
        <div class="nav-a"><a href="javascript:window.scrollToId('engineering')">Engineering</a></div>
        <div class="nav-a"><a href="javascript:window.scrollToId('success')">Engineering Success</a></div>
      </div>
    </nav>
    
    <main>
      <div class="background-image-wrapper">
        <img class="background-image" src="https://2020.igem.org/wiki/images/1/1f/T--Korea-SIS--background.png"/>
      </div>      
      <div class="main-page">
        <div class="main-page-inner-with-nav">
          <h1>Engineering</h1>
          <h2 class="green-h1" id='method'>Method of Promotion</h2>
          <div class="paragraph">
            <div class="image-wrapper" style="flex-direction: column;">
              <img width="800px" src="https://2020.igem.org/wiki/images/2/21/T--Korea-SIS--PromotionMethod.png"/>
              <span>Figure 1. Method of promotion</span>
            </div>
            <div>
              To make the recombination <i>E.coli</i> that expresses CYP1A2, which is a human protein that we need for measuring the concentration of toxin, we had to design the CYP1A2 expression system. So we searched for an appropriate vector plasmid that has multiple cloning sites(MCS) and the antibiotics resistance gene(Here we have kanamycin resistance gene.) for our system. By using two restriction enzymes(Here we’re going to use <i>EcoRI</i> and and <i>BamHI</i>.) and two primers, recombinant vectors that carry the CYP1A2 gene can be constructed. Then, this vector plasmid has to be taken up to the <i>E.coli</i> cell, especially bacterial competent cells that have been specially treated to transform efficiently. 
            </div>
            <div>
              To confirm whether the vectors with CYP1A2, our target gene are transferred to <i>E.coli</i> cells, the resistance gene that vectors have can be used. Our vectors have a kanamycin resistance gene, so kanamycin treated media will be used to get only <i>E.coli</i> cells which have vectors that are carrying the target gene. 
            </div>
            <div>
              After that, Recombinant <i>E.coli</i> will be grown in the 37C, and they will express CYP1A2 protein. Protein extraction process and protein purification can help to get the enzyme, without other molecules in the total cell lysate. And then, now we can use the enzyme to treat the crops, which have aflatoxins that should be measured by a spectrophotometer.
            </div>
            <div>
              For protein purification, we are going to use His-tagged CYP1A2 gene.
            </div>
            <div class="image-wrapper" style="flex-direction: column;">
              <img width="800px" src="https://2020.igem.org/wiki/images/a/a9/T--Korea-SIS--expression.png">
              <div style="margin-left: -20rem;">
                <div>vector: pcDNA 3.1 </div>
                <div>promoter region: CMV promoter</div>
              </div>
              <span><br>Figure 2. CYP1A2 expression system in <i><i>E.coli</i></i></span>
            </div>
            <div class="image-wrapper" style="flex-direction: column;">
              <img width="700px" src="https://2020.igem.org/wiki/images/7/7d/T--Korea-SIS--3_2_design.png">
              <span><br>Figure 3. Vector map of CYP1A2 expression system</span>
            </div>
          </div>

          <h2 class="green-h1" id='engineering'>Engineering</h2>
          <div class="paragraph">
            <ul>
              <li>
                <h2>
                  <i>E.coli</i> transformation
                </h2>
              </li>
            </ul>
            <ol>
              <li>
                Take competent cells out of -80°C and thaw on ice (approximately 20-30 mins).
              </li>
              <li>
                Incubate LB agar plates in 37°C incubator.
              </li>
              <li>
                Mix 50ng of DNA into 20μL of competent cells in a tube. <br>
                Gently mix by flicking the bottom of the tube with your finger a few times.
              </li>
              <li>
                Incubate the competent cell/DNA mixture on ice for 20-30 mins.
              </li>
              <li>
                Heat shock each transformation tube by placing the bottom 1/2 to 2/3 of the tube into a 42°C water bath for 30-60 secs (45 secs is usually ideal, but this varies depending on the competent cells you are using).
              </li>
              <li>
                Put the tubes back on ice for 2 min.
              </li>
              <li>
                Add 250-1,000 μl LB or SOC media (without antibiotic) to the bacteria and grow in a 37°C shaking incubator for 45 min.
              </li>
              <li>
                Plate some or all of the transformation onto a 10 cm LB agar plate containing the appropriate antibiotic.
              </li>
              <li>
                Incubate plates at 37°C overnight.
              </li>
              <li>
                Grow cells in the LB media. (liquid)
              </li>
            </ol>
          </div>
          <div class="paragraph">
            <ul>
              <li>
                <h2>
                  Protein extraction 
                </h2>
              </li>
            </ul>
            <ol>
              <li>
                Centrifuge the media at 13000rpm, 4C for 5 min. 
              </li>
              <li>
                Discard the supernatant, and wash with PBS. 
              </li>
              <li>
                Centrifuge in 13000rpm, 4°C for 1 min and discard the supernatant. 
              </li>
              <li>
                Put the lysis buffer(B-PER) to the pellet, and mix them with pipetting.
              </li>
              <li>
                After incubation(1hr, room temperature), centrifuge in 13000rpm, 4°C for 1 min.
              </li>
              <li>
                Transfer the supernants to new e-tubes that are labeled.
              </li>
            </ol>
          </div>
          <div class="paragraph">
            <ul>
              <li>
                <h2>Protein Purification</h2>
              </li>
            </ul>
            <div>
              <h3>1) Preparing Ni-NTA Column</h3>
              <ol>
                <li>
                  Prepare Native Binding Buffer.
                  <div style="margin-bottom: 10px;">
                    ①without imidazole<br>
                    : Use 30 mL of the 1X Native Purification Buffer for use as the Native Binding Buffer (used for column preparation, cell lysis, and binding).<br><br>
                    ②with imidazole<br>
                    : To prepare 30 mL Native Binding Buffer with 10 mM imidazole, combine:<br>
                    • 30 mL of 1X Native Purification Buffer<br>
                    • 100 μL of 3 M Imidazole, pH 6.0<br>
                    Mix well and adjust pH to 8.0 with NaOH or HCl.
                </li>
                <li>
                  Resuspend the Ni-NTA Agarose in its bottle by inverting and gently tapping the bottle repeatedly.
                </li>
                <li>
                  Pipet or pour 1.5 mL of the resin into a 10mL Purification Column. Allow the resin to settle completely by gravity for 10 minutes. Gently aspirate the supernatant.
                </li>
                <li>
                  Add 6 mL sterile, distilled water and resuspend the resin by alternately inverting and gently tapping the column.
                </li>
                <li>
                  Allow the resin to settle using gravity or centrifugation as in Step 2, and gently aspirate the supernatant.
                </li>
                <li>
                  For purification under Native Conditions, add 6 mL Native Binding Buffer 
                </li>
                <li>
                  Resuspend the resin by alternately inverting and gently tapping the column.
                </li>
                <li>
                  Allow the resin to settle using gravity or centrifugation as described in Step 2, and gently aspirate the supernatant.
                </li>
                <li>
                  Repeat Steps 5 through 7.
                </li>
              </ol>
            </div>

            <div>
              <h3>2) Purification</h3>
              <ol>
                <li>
                  To prepare 50 mL Native Wash Buffer with 20 mM imidazole, combine:
                  <ul>
                    <li>
                      50 mL of 1X Native Purification Buffer
                    </li>
                    <li>
                      335 μL of 3 M Imidazole, pH 6.0 <br>
                      Mix well and adjust pH to 8.0 with NaOH or HCl.
                    </li>
                  </ul>
                </li>
                <li>
                  Add 8 mL lysate prepared under native conditions to a prepared Purification Column.
                </li>
                <li>
                  Bind for 60 minutes using gentle agitation to keep the resin suspended in the lysate solution.
                </li>
                <li>
                  Settle the resin by gravity or low-speed centrifugation (800 g), and carefully aspirate the supernatant. Save supernatant at 4°C for SDS-PAGE analysis.
                </li>
                <li>
                  Wash with 8 mL Native Wash Buffer. Settle the resin by gravity or low-speed 
                </li>
                <li>
                  Repeat Step 4 three more times.
                </li>
                <li>
                  Prepare Native Elution Buffer.
                </li>
                <li>
                  To prepare 15 mL Native Elution Buffer with 250 mM imidazole, combine:
                  <ul>
                    <li>
                      13.75 mL of 1X Native Purification Buffer
                    </li>
                    <li>
                      1.25 mL of 3 M Imidazole, pH 6.0 <br>
                      Mix well and adjust pH to 8.0 with NaOH or HCl.
                    </li>
                  </ul>
                </li>
                <li>
                  Clamp the column in a vertical position and snap off the cap on the lower end. Elute the protein with 10mL Native Elution Buffer Collect 1 mL fractions and analyze with SDS-PAGE.
                </li>
              </ol>    
            </div>
          </div>

          <h2 class="green-h1" id="success">Engineering success</h2>
          <div>
            To deal with unexpected results, we’ll get samples of each step in procedure.<br>
            Then we can explain when we’ve made some mistakes in experiments.
          </div>
          <div class="paragraph">
            <h3>I) Poor resolution, purity</h3>
            <ul>
              <li>
                Column poorly packed
              </li>
              <li>
                Elution conditions not optimal: gradient, flow rate control
              </li>
              <li>
                Protein precipitated in column: cleaning procedures are needed.
              </li>
              <ul>
                <li>
                  Ex. HIC(Reduce salt concentration in buffer), IEX(Modify buffer, pH, salt conditions during run to maintain stability)
                </li>
              </ul>
            </ul>
          </div>

          <div class="paragraph">
            <h3>
              II) Influence of gene and/or protein sequence on expression and solubility
            </h3>
            <div>
              One of the most common reasons foreign proteins fail to express is the presence of "cruel" codons in the target mRNA. However, codon bias can be ruptured by codon-optimized gene synthesis. One advantage of gene synthesis is the ability to alter the codon bias of a gene and a more compatible host. In the case of E. coli, expression complemented by a rare tRNA can break the codon bias of the gene.
            </div>
            <div>
              The probability of successful expression of the expressed protein is as the molecular weight increases, especially in the case of proteins exceeding 60 kD. When using E. In E. coli, it is advantageous to design the structure of the phosphorus protein region rather than the full-length protein and use a fusion tag to improve solubility. Tags, which are very helpful in protein purification, have little adverse effect on biological or biochemical activity.
            </div>
            <div>
              The starting and ending residues of the target area can also affect the expression yield and solubility. The optimal boundary of the protein domain composition should be determined using the available functional and structural data of the protein. For proteins with unknown domain structures, stitching the sequence of the target protein into the homolocoprotein structure can help to determine the optimal domain boundary. When the protein structure of the homoeum is not available, prediction of secondary structural elements should be used.
            </div>
          </div>
            
          <div class="paragraph">
            <h3>
              III) Influence of vector on expression and solubility
            </h3>
            <div>
              DNA sequence elements that direct the transcription and translation of a target gene include promoters, regulatory sequences, Shine-Dalgarno boxes, transcription terminators, and origins of replication. In addition, the expression vector contains selection elements that support plasmid selection within the host cell. Another important feature of the E. coli expression vector is the presence of a fusion tag.
            </div>
            <div>
              When choosing a promoter system, consider the nature of the protein target and the desired downstream use. If the protein target is a toxic protein, consider using a promoter system with extremely low basal expression. Alternatively, a strong promoter should be selected for maximum protein yield. For proteins that are easy to accumulate, cold shock accelerators that are expressed at low temperatures can be tested.
            </div>
            <div>
              Larger bacteria and isomeric proteins tend to fold and aggregate more slowly. To prevent the accumulation of E. coli and facilitate folding, a protein protective plate and a folding catalyst can be used. The protein of interest can be co-expressed with a second protein encoded on the same plasmid or on a separate plasmid.
            </div>
            <div>
              Fusion tags are genetically fused to the protein of interest to increase protein solubility. It is often necessary to test multiple fusion tags to determine which tag yields the maximum yield of a water-soluble protein. The placement of the tag on either the N-terminal or C-terminal of the target protein is also important. N-terminal melting agents are the most common and have the advantage of successfully enhancing the expression of water-soluble proteins than C-terminal melting agents.
            </div>
            <div>
              Since the presence of the fusion tag may interfere with the biological activity of the recombinantly expressed protein, it may be important to enzymatically remove the tag after the fusion protein is purified. It is recommended to include the cracked portion of the protease by sequence so that the tag can be removed.
            </div>

          </div>
          
          <div class="paragraph">
            <h3>
              IV) Influence of host strains on expression of heterologous proteins
            </h3>

            <div>
              Bacterial host strains have been developed to support the expression of foreign proteins. Commercially available E. coli species are specially designed for the specific expression of proteins that are susceptible to proteolysis, contain rare codons, or require disulfide compounds. For proteins susceptible to proteolysis, the use of protease-deficient strains such as E. coli BL21 or its derivatives is recommended.
            </div>
            <div>
              Differences in codon frequency between the target gene and the expressing host can lead to translational delays, premature translation termination, and amino acid mismatch. This difference can be overcome by supplying rare tRNAs during expression. Bacterial strains containing plasmids encoding rare tRNAs should be used to promote efficient expression of genes containing high frequencies of rare codons.
            </div>
            <div>
              Differences in codon frequency between the target gene and the expression host can lead to translational stalling, premature translation termination, and amino acid mis-incorporation. This difference may be overcome by supplying the rare tRNAs during expression. Bacterial strains that contain plasmids that encode rare tRNAs should be used to promote the efficient expression of genes that contain high frequencies of rare codons.
            </div>
            <div>
              For proteins containing disulfide bonds, expression of thioredoxin reductase and glutathione reductase host strains aids in the formation of cytoplasmic disulfide bonds and enhances the solubility of folded disulfide content proteins. An alternative to expressing disulfide-containing proteins is to target proteins expressed in E. coli blood vessels, which promote the formation of disulfide bonds because of their high oxidative properties.
            </div>
 
          </div>

          <div class="paragraph">
            <h3>
              V) Improving solubility of proteins by changing expression conditions
            </h3>
            <div>
              The use of strong expression promoters and high inducers can lead to high protein concentrations that can cause proteins to aggregate prior to folding. Reducing the rate of transcription and/or translation will facilitate folding because the newly synthesized protein can fold before it can be aggregated. The following are general expression condition parameters that can be manipulated to increase protein solubility.
            </div>
            <div>
              Temperature: Lowering the expression temperature (15-25°C) improves the solubility of the re-expressed protein. At lower temperatures, cellular processes slow down, leading to reduced transcription, translation, cell division, and protein aggregation. Lowering the expression temperature also reduces the degradation of proteolytic sensitive proteins.
            </div>
            <div>
              Concentration of derivatives: When the concentration of the inducer is lowered, the transcription rate is lowered, thereby improving the solubility and activity of the recombinant protein.
            </div>
            <div>
              Media Selection: Batch culture is the most common method of culturing cells for expression of recombinant proteins. All nutrients needed for growth should be included in the medium and supplied from the beginning.
            </div>

          </div>

          <div class="paragraph">
            <h3>
              VI) Improving protein purification
            </h3>
              <div>
                Protein is dissolved and purified in a well-filled solution containing an ionic strength equivalent to 500 mM monovalent acid salt such as NaCl.
              </div>
              <div>
                Use Immobilized Metal Affinity Chromatography (IMAC) as an initial purification step.
              </div>
              <div>
                If further purification is required, use size exclusion chromatography (gel filtration). If necessary, use ion exchange chromatography as the final'brightening' step.
              </div>
              <div>
                The affinity tag can be removed to minimize the non-genic sequence of the recombinant protein and to achieve further purification.
              </div>
          </div>

        </div>
      </div>
    </main>
    
    <footer>
      <div class="footer-image-wrapper">
        <img src="https://2020.igem.org/wiki/images/0/0d/T--Korea-SIS--footer.png" />
      </div>
    </footer>
    <script type="text/javascript" src="https://2020.igem.org/wiki/index.php?title=Template:Korea-SIS/JS&action=raw&ctype=text/javascript"></script>
  </body>
</html>